src/page/gjennomforing-detaljer/GjennomforingDetaljerPage.tsx

```diff
@@ -32,40 +41,40 @@ export const GjennomforingDetaljerPage = () : React.ReactElement => {
 		<main className={styles.page}>
 			<Tilbakelenke to={FORSIDE_PAGE_ROUTE} className={globalStyles.blokkM}/>
 
-			<Heading size="medium" spacing>{gjennomforing.navn}</Heading>
-
-			<GjennomforingGenerellInfo gjennomforing={gjennomforing}/>
+			<div className={styles.heading}>
+				<Heading size="medium" spacing>{gjennomforing.navn}</Heading>
+
+				<Show if={isNotStartedOrPending(fjernFraOversiktenPromise)}>
```

Comment:
Kan være en fordel å splitte ting opp. Her er det mange ting relatert til fjerning av gjennomføring som må ses i sammenhent

Reply:
👍 Terskelen min for å splitte ut ting er kanskje litt mindre kritisk, men er enig at det blir litt mer leselig.